<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="xRotSlider">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>610</y>
      <width>981</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="yRotSlider">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>630</y>
      <width>981</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="zRotSlider">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>650</y>
      <width>981</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>610</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X Rotation Speed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>630</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y Rotation Speed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>650</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Z Rotation Speed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toggleAnimationPushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>630</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Toggle Animation</string>
    </property>
   </widget>
   <widget class="QSlider" name="redSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="blueSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="greenSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="alphaSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="redSliderLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Red Slider</string>
    </property>
   </widget>
   <widget class="QLabel" name="alphaSliderLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Alpha Slider (disabled)</string>
    </property>
   </widget>
   <widget class="QLabel" name="greenSliderLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Green Slider</string>
    </property>
   </widget>
   <widget class="QLabel" name="blueSliderLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue Slider</string>
    </property>
   </widget>
   <widget class="QComboBox" name="shapeComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Cube</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pyramid</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tetrahedron</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Octahedron</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="shapeComboBoxLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Shape</string>
    </property>
   </widget>
   <widget class="QLabel" name="surfaceSlidersLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Surface Sliders</string>
    </property>
   </widget>
   <widget class="QLabel" name="greenEdgeSliderLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Green Slider</string>
    </property>
   </widget>
   <widget class="QSlider" name="greenEdgeSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="blueEdgeSliderLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue Slider</string>
    </property>
   </widget>
   <widget class="QLabel" name="redEdgeSliderLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Red Slider</string>
    </property>
   </widget>
   <widget class="QLabel" name="edgeSlidersLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Edge Sliders</string>
    </property>
   </widget>
   <widget class="QSlider" name="blueEdgeSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="alphaEdgeSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="alphaEdgeSliderLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Alpha Slider (disabled)</string>
    </property>
   </widget>
   <widget class="QSlider" name="redEdgeSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="extrasLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Extras</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rainbowModeRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Rainbow Mode</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Upload Texture</string>
    </property>
    <addaction name="actionJk_I_ran_out_of_time"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFrom_this_pc">
   <property name="text">
    <string>From this pc...</string>
   </property>
  </action>
  <action name="actionJk_I_ran_out_of_time">
   <property name="text">
    <string>I ran out of time :(</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
